src/Integration.UnitTests/Binding/DotNetBindingConfigProviderTests.cs

```diff
@@ -79,7 +79,7 @@ public void GetRules_UnsupportedLanguage_Throws()
         }
 
         [TestMethod]
-        public void IsSupported()
+        public void IsLanguageSupported()
```

Comment:
Miscellaneous: renamed to match the method. I should have made the method `static` while I was at it...